apps/cow-tools/codegen.ts

```diff
@@ -3,9 +3,9 @@ import type { CodegenConfig } from "@graphql-codegen/cli";
 const config: CodegenConfig = {
   overwrite: true,
   schema: "https://milkman-api.up.railway.app/",
-  documents: "src/**/*.tsx",
+  documents: "src/**/*.ts",
   generates: {
-    "src/gql/generated.ts": {
+    "src/lib/gql/generated.ts": {
```

Comment:
why is this needed?

Reply:
one query was on .ts file, that wasn't watched by the codegen. Also, the path for the generated file was wrong. Regarding the .ts and .tsx I think that I can add both here.